src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -673,7 +674,21 @@ private BoundExpression MakeImplicitConversion(BoundExpression rewrittenOperand,
                     Symbol.ReportUseSiteDiagnostic(useSiteInfo, _diagnostics, syntax.Location);
                 }
                 var fieldAccess = MakeTupleFieldAccess(syntax, field, savedTuple, null, LookupResultKind.Empty);
-                var convertedFieldAccess = MakeConversion(fieldAccess, destElementTypes[i], @checked);
+
+                Conversion conversion;
+
+                HashSet<DiagnosticInfo> useSiteDiagnostics = null;
+                if (explicitCastInCode)
+                {
+                    conversion = this._compilation.Conversions.ClassifyConversionForCast(fieldAccess.Type, destElementTypes[i], ref useSiteDiagnostics);
```

Comment:
Yes, eventually. There will be a separate change specifically to reduce cases of conversion classification in the lowering. I did not want to do just a part of it here.